src/Compilers/VisualBasic/Portable/Symbols/Source/SourcePropertyClonedParameterSymbolForAccessors.vb

```diff
@@ -0,0 +1,37 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    Friend NotInheritable Class SourcePropertyClonedParameterSymbolForAccessors
+        Inherits SourceClonedParameterSymbol
+
+        Public Sub New(originalParam As SourceParameterSymbol, newOwner As MethodSymbol)
+            MyBase.New(originalParam, newOwner, originalParam.Ordinal)
+        End Sub
+
+        Friend Overrides ReadOnly Property IsCallerLineNumber As Boolean
+            Get
+                Return _originalParam.IsCallerLineNumber
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsCallerMemberName As Boolean
+            Get
+                Return _originalParam.IsCallerMemberName
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsCallerFilePath As Boolean
+            Get
+                Return _originalParam.IsCallerFilePath
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property CallerArgumentExpressionParameterIndex As Integer
+            Get
+                Return _originalParam.CallerArgumentExpressionParameterIndex
```

Comment:
>_originalParam.CallerArgumentExpressionParameterIndex [](http://example.com/codeflow?start=23&length=53)

Is there a unit-test that hits this code path?

Reply:
None of the current tests hits this path. :(
It *could* be unreachable, but I can't tell for sure. Will give it more time and see what I can get.

Reply:
I believe this is unreachable. The parameters used in the callsite comes from `OverloadResolution.Candidate`. There are only two possible candidates:

https://github.com/dotnet/roslyn/blob/f6ba23211d9272426214fe055ab8f1e9983e6dd4/src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb#L144-L145

https://github.com/dotnet/roslyn/blob/f6ba23211d9272426214fe055ab8f1e9983e6dd4/src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb#L362-L363

So for this path, we're always accessing `.Parameters` of properties, not accessors. There is a test for properties, so I'll go ahead and throw unreachable here.